client/header/activity-panel/panels/reviews.js

```diff
@@ -93,34 +93,22 @@ class ReviewsPanel extends Component {
 			</div>
 		);
 
-		const cardActions = () => {
-			const mainLabel =
-				'approved' === review.status
-					? __( 'Unapprove', 'woocommerce-admin' )
-					: __( 'Approve', 'woocommerce-admin' );
-			return (
-				<SplitButton
-					mainLabel={ mainLabel }
-					menuLabel={ __( 'Select an action', 'woocommerce-admin' ) }
-					onClick={ noop }
-					controls={ [
-						{
-							label: __( 'Reply', 'woocommerce-admin' ),
-							onClick: noop,
-						},
-						{
-							label: __( 'Spam', 'woocommerce-admin' ),
-							onClick: noop,
-						},
-						{
-							label: __( 'Trash', 'woocommerce-admin' ),
-							onClick: noop,
-						},
-					] }
-				/>
-			);
+		const manageReviewEvent = {
+			date: review.date_created_gmt,
+			review: review.id,
```

Comment:
I think we can omit the `id` property. It does't seem useful from a Tracks perspective.